build.gradle

```diff
@@ -22,6 +22,9 @@ group "io.kestra.plugin"
 
 subprojects {
     group "io.kestra.plugin"
+
+    apply plugin: "java-library"
+
```

Comment:
I'm not sure it's correct to move it there

Reply:
I was unable to build plugin-script as it needed to apply java plugin before specifying the target / source compatibility (I had error where it said that dependency script is not built for proper java version)